aiida_mlip/calculations/geomopt.py

```diff
@@ -44,49 +44,44 @@ def define(cls, spec: CalcJobProcessSpec) -> None:
         """
         super().define(spec)
 
-        # Additional inputs for geometry optimization
+        # Additional inputs for geometry optimisation
         spec.input(
             "traj",
             valid_type=Str,
             required=False,
-            default=lambda: Str(cls._DEFAULT_TRAJ_FILE),
-            help="Path to save optimization frames to",
+            default=lambda: Str(cls.DEFAULT_TRAJ_FILE),
+            help="Path to save optimisation frames to",
         )
         spec.input(
```

Comment:
no default?

Reply:
I'll write something in the documentation to exaplain how it works at the moment and open another issue to fix some things with the output in the future